pipelines/_ingredients/LSSTCam/DRP.yaml

```diff
@@ -7,8 +7,11 @@ imports:
   - $ANALYSIS_TOOLS_DIR/pipelines/coaddQualityExtended.yaml
   - $ANALYSIS_TOOLS_DIR/pipelines/diaTractQualityCore.yaml
   - $ANALYSIS_TOOLS_DIR/pipelines/matchedVisitQualityCore.yaml
-  - $ANALYSIS_TOOLS_DIR/pipelines/visitQualityCore.yaml
+  - location: $ANALYSIS_TOOLS_DIR/pipelines/visitQualityCore.yaml
+    exclude:
+      - analyzeCalibrateMetadata
```

Comment:
If you keep excluding this everywhere, it seems to tell me that maybe this does not believe in the visitQualityCore, as that is defined explicitly as the tasks we want to always run

Reply:
I'll delete it on [DM-47632](https://rubinobs.atlassian.net/browse/DM-47632), and in fact I had originally deleted it on an earlier version of this ticket before I realized I needed to keep ComCam using the old tasks for a bit longer.  I just need to keep it as long as any instrument is still using `calibrate`.